src/main/java/org/springframework/social/salesforce/api/impl/ApiTemplate.java

```diff
@@ -36,7 +43,47 @@ public ApiTemplate(Salesforce api, RestTemplate restTemplate) {
     @SuppressWarnings("unchecked")
     public Map<String, String> getServices(String version) {
         requireAuthorization();
-        return restTemplate.getForObject(api.getBaseUrl() + "/v{version}", Map.class, version);
+        return restTemplate.getForObject(api.getBaseUrl() + "/{version}", Map.class, version);
+    }
+
+    @SuppressWarnings("unchecked")
+    public Map<String, String> getServices() {
+        requireAuthorization();
+        return this.getServices(getVersion());
+    }
+
+    public void setVersion(String version)
+    {
+        if (StringUtils.isNotBlank(version) && validateVersionString(version))
```

Comment:
Perhaps we should throw an exception here if validation fails rather than failing silently?

(The test would have to be updated as well of course)